recipe-server/normandy/base/decorators.py

```diff
@@ -25,3 +27,23 @@ def wrapped_view(*args, **kwargs):
         with transaction.atomic(), revisions.create_revision(manage_manually=True):
             return view_func(*args, **kwargs)
     return wraps(view_func)(wrapped_view)
+
+
+def api_cache_control(**kwargs):
+    """
+    Adds cache headers to a view using our API cache header defaults.
+    """
+    if settings.API_CACHE_ENABLED:
+        directives = {
+            'public': True,
+            'max_age': settings.API_CACHE_TIME,
+        }
+    else:
+        directives = {
+            'no_cache': True,
+            'no_store': True,
+            'must_revalidate': True,
+        }
+
+    directives.update(kwargs)
```

Comment:
I think that doing this in all cases will cause problems when `API_CACHE_ENABLED=False`. The actions view would still be cache, even if the cache is disabled, right?